man/resolved.conf.xml

```diff
@@ -317,6 +317,17 @@
         <xi:include href="version-info.xml" xpointer="v232"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RefuseRecordTypes=</varname></term>
+        <listitem><para>Takes a string argument separated by spaces. This option is to refuse (deny) record types
+         set from from being queried</para>
```

Comment:
Please write something like the following:
`Takes a list of DNS record types separated by space. Specified record types in each query will be refused. This option can be specified multiple times. If an empty string is specified, then all previous assignments are cleared.`

Reply:
@yuwata done.